CONTRIBUTING.md

```diff
@@ -10,6 +10,14 @@
 
 ## Maintainers
 
+* Note: To publish a new release you need to get access to the S3 bucket ...
+  * Contact one of the existing commiters/maintainers to get the aws keys
+  * Add the following section to your `~/.aws/credentials` file ...
+    ```
+    [nmesos]
+    aws_access_key_id = <access>
+    aws_secret_access_key = <secret>    
+    ```
```

Comment:
Just to document what you need to do to setup the aws profile with the right credentials.